packages/blade/src/components/Spinner/_decisions/decisions.md

```diff
@@ -0,0 +1,62 @@
+# Spinner <!-- omit in toc -->
+A spinner is an element with a looping animation that indicates loading is in process.
+
+<img src="./spinner-thumbnail.png" width="150" />
+
+## Design
+- [Figma - Spinner](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=14825%3A203592)
+
+
+## BaseSpinner
+A BaseSpinner will contain some extra props like `intent`. This will not be exposed to the consumer and will only be used for internal components.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| intent | `positive`,`negative`, `notice`, `information`, `neutral` | `undefinedg` | The intent of the spinner to be rendered. |  |
+| contrast | `low`, `high` | `low` | The contrast of the spinner to be rendered. |  |
+| size | `small`, `medium`, `large` | `medium` | The size of the spinner to be rendered. |  |
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the spinner. |
+
+
+## Spinner
+A wrapper around Spinner which will be exposed to the consumer to be used as a Spinner. This component will not allow passing `intent` to the Spinner.
+
+### API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| contrast | `low`, `high` | `low` | The contrast of the spinner to be rendered. |  |
+| size | `small`, `medium`, `large` | `medium` | The size of the spinner to be rendered. |  |
+| accessibilityLabel | `string` | `"Loading"` | The accessibility label (aria-label) for the spinner. |
+
+### Sample Usage
+```jsx
+import { Spinner } from '@razorpay/components';
+
+<Spinner
+  contrast='high'
+  size='large' 
+  accessibilityLabel='Loading'
+/>
+```
+
+### Accessibility
+- We'll accept a prop `accessibilityLabel` which will be passed on as `aria-label` for web & `accessibilityLabel` for native to aide screen readers
+- By default the value of `accessibilityLabel` will be `Loading` and can be overwritten with something more meaningful by the consumer
+- We will also set `aria-live` as `assertive` to inform a user with a screen-reader that something is loading
+
+
+## Open Questions
+- Should we set `aria-role` as `alert` for accessibility? [Carbon DS](https://carbondesignsystem.com/components/loading/accessibility#how-it-works) does this as well.
```

Comment:
Might need to do some research on this but can we instead use aria busy? alert doesn't sound like the best thing (this is not exactly an alert - and probably doesn't need to be this assertive). 

Regardless I think aria-live is a good idea to announce changes in label. Also can consider aria-hidden for the animating component.

Reply:
Yeah I'll look some more into what makes sense for a11y here. We wouldn't need additional element with aria-hidden if we have aria-label already right?